package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --mode production"
+    "serve": "webpack serve --mode development"
   },
```

Comment:
- [ ] Nice work setting up webpack server :clap: However I noticed that you remove the build command. Kindly add it back because it help to build for production by coping all the file to the `dist` folder since it not allow to create files manually in it. Therefore it would be nice to have both command (`build` and `serve`) in your script.